File: fibo/FibonacciSeries.java
public class FibonacciSeries {
    public static void main(String[] args) {
        int n = 10; // Number of Fibonacci numbers to generate
        generateFibonacci(n);
    }

    public static void generateFibonacci(int n) {
        long first = 0;
        long second = 1;

        System.out.println("Fibonacci Series (First " + n + " numbers):");

        for (int i = 0; i < n; i++) {
            System.out.print(first + " ");
            long next = first + second;
            first = second;
            second = next;
        }
    }
}

File: fibo1/fibonacciSeries.py
# Function to generate a Fibonacci sequence
def generate_fibonacci(n):
    # Initialize the first two Fibonacci numbers
    a, b = 0, 1
    
    # Check if the input is valid
    if n <= 0:
        return []
    elif n == 1:
        return [a]
    elif n == 2:
        return [a, b]
    
    # Generate the Fibonacci sequence
    fibonacci_sequence = [a, b]
    for _ in range(2, n):
        a, b = b, a + b
        fibonacci_sequence.append(b)
    
    return fibonacci_sequence

# Number of Fibonacci numbers to generate
n = 10

# Generate and print the Fibonacci sequence
fib_sequence = generate_fibonacci(n)
print(f"Fibonacci sequence of {n} numbers: {fib_sequence}")
